mailbox/api/src/test/java/org/apache/james/mailbox/MailboxManagerTest.java

```diff
@@ -144,7 +144,7 @@ void tearDown() throws Exception {
     }
 
     @Test
-    public void creatingConcurrentlyMailboxesWithSameParentShouldNotFail() throws Exception {
+    protected void creatingConcurrentlyMailboxesWithSameParentShouldNotFail() throws Exception {
```

Comment:
nothing at all?

Reply:
Doesn't work with nothing at all, as the test is being disabled on `JPAMailboxManagerTest` inheriting `MailboxManagerTest`